packages/stark-core/src/logging/services/logging.service.spec.ts

```diff
@@ -230,7 +257,15 @@ describe("Service: StarkLoggingService", () => {
 				expect(message.message).toContain("dummy error message");
 				expect(message.timestamp).toBeDefined();
 				expect(message.error).toBeDefined();
-				expect(message.error).toContain("this is the error");
+
+				const starkError: StarkError = (<StarkError>message.error);				
+				if (starkError) {
```

Comment:
Same here, we could replace the if block with an expectation (see my previous comment)